recipe/meta.yaml

```
@@ -30,21 +32,37 @@ outputs:
         - python
         - pip
         - setuptools
```

```suggestion
        - setuptools >=80
        - setuptools-scm >=8
```
https://github.com/jcrist/msgspec/blob/0.20.0/pyproject.toml#L3-L4

I think it will work without the patch

---

A bit different, patch is about another problem and can't be removed until msvc 2022 available

About setuptools-scm
Previous pkg builder took sdist from github to having tests inside, but that sdist doesn't work correctly with setuptools-scm . So if we want a build over setuptools-scm need to change sdist to PYPI instead of github.

---

Sorry for misleading. I meant without `echo "__version__ = \"{{ version }}\"" > src/msgspec/_version.py`

---

If we want to get read of this
`echo "__version__ = \"{{ version }}\"" > src/msgspec/_version.py
 `
 have to change sources to PYPI

---

Even with github sources, `setuptools-scm` will create `msgspec/_version.py` during `pip install .`

---

I don't have this issue in my draft PR https://github.com/AnacondaRecipes/msgspec-feedstock/pull/4